### PR TITLE
Custom title and output path

### PR DESCRIPTION
**Pull request by mgerasimchuk**
…go-coverage-report/issues/10

### CODE REVIEW

**Comment by mgerasimchuk** [comment]
Have tested on my own projects:
- jobs:
  - https://github.com/mgerasimchuk/protty/blob/master/.github/workflows/test-unit.yml#L11-L18
  - https://github.com/mgerasimchuk/protty/blob/master/.github/workflows/test-integration.yml#L11-L18
- wiki: https://github.com/mgerasimchuk/protty/wiki/Test-coverage-report
- readme: https://github.com/mgerasimchuk/protty/blob/master/README.md

*sorry for the unrelated changes in one MR, but I also added a possibility to change a badge title

File: action.yml [hunk 1]
```diff
@@ -10,9 +10,14 @@ inputs:
   badge-style:
     description: Badge style generated by shields.io (one of flat, flat-square, plastic, for-the-badge, social).
     default: flat
+  badge-title:
+    description: Badge title generated by shields.io.
+    default: coverage
```
Comment (ncruces) [inline action.yml:15]: 👍

File: action.yml [hunk 2]
```diff
@@ -10,9 +10,14 @@ inputs:
...
   report:
     description: Generate an HTML coverage report.
     default: true
+  report-path:
+    description: Custom path for an HTML coverage report.
```
Comment (ncruces) [inline action.yml]: This isn't really a the report path.
It's actually the output folder where _all_ output files are created, right?

**Review by ncruces — COMMENTED** [review]
Thanks for the PR!
Tbh I'm not 100% convinced about this…

File: action.yml (hunk 2 above)
Comment (mgerasimchuk) [reply action.yml]: Yes, actually it is, I just didn't want to add something new, but if you have a suggestions, I'll fix it..

Anyway, I can rename param and ENVs like:
```
output-path:
    description: Custom path for an HTML coverage report and other results file like svg badge and chart.
```

What do you think?

Comment (mgerasimchuk) [reply action.yml]: fixed

File: action.yml [hunk 3]
```diff
@@ -10,9 +10,14 @@ inputs:
   badge-style:
     description: Badge style generated by shields.io (one of flat, flat-square, plastic, for-the-badge, social).
     default: flat
+  badge-title:
+    description: Badge title generated by shields.io.
+    default: coverage
   report:
     description: Generate an HTML coverage report.
     default: true
+  output-path:
```
Comment (ncruces) [inline action.yml:19]: I'll probably change this to `output-dir`, for extra clarity.

File: action.yml [hunk 4]
```diff
@@ -61,5 +67,5 @@ runs:
         git config --local user.name  "GitHub Action"
         git config --local user.email "action@github.com"
         test ${{inputs.amend}} == true   && \
-          git commit --amend --no-edit   && git push --force-with-lease || \
-          git commit -m"Update coverage" && git push
+          git commit --amend --no-edit   && for i in {1..10}; do git push --force-with-lease && c=0 && break || c=$? && git pull --rebase; done; (exit $c) || \
```
Comment (ncruces) [inline action.yml:70]: Love the retry logic… but it's getting to the point that I need to create a proper shell script, I guess.

Will do it as a follow up.

**Review by ncruces — APPROVED** [review]

**Pull request merged**